fix/jedi/convinfo.rrfs

```diff
@@ -228,7 +228,7 @@
  mtacld   183    0    0     1.5    7.0   5.6   1.3 N/A
  mtacld   187    0    0     1.5    7.0   5.6   1.3 N/A
  gosctp   151    0    0     1.5    7.0   5.6   1.3 N/A
- radref     0    0    0     1.5    7.0   5.6   1.3 N/A
+ reflectivity     0    0    0     1.5    7.0   5.6   1.3 N/A
```

Comment:
I would suggest we use `refl`,  `refl10`, or `refl10cm`

Reply:
@hu5970 We already have `dbz` from rrfs.v1, see line 234

Reply:
change to refl10cm